src/Test/Runner/Log.elm

```diff
@@ -0,0 +1,76 @@
+module Test.Runner.Log exposing (run, runWithOptions)
+
+{-| # Log Runner
+
+Runs a test and outputs its results using `Debug.log`, then calls `Debug.crash`
+if there are any failures.
+
+This is not the prettiest runner, but it is simple and cross-platform. For
+example, you can use it as a crude Node runner like so:
+
+    $ elm-make LogRunnerExample.elm --output=elm.js
+    $ node elm.js
+
+This will log the test results to the console, then exit with exit code 0
+if the tests all passed, and 1 if any failed.
+
+@docs run, runWithOptions
+-}
+
+import Random
+import Test exposing (Test)
+import Test.Runner.String exposing (Summary)
+import String
+
+
+{-| Run the test using the default `Test.Runner.String` options.
+-}
+run : Test -> a -> a
```

Comment:
I would expect this to be `Test -> Task x ()`. Right now folks are using `Debug.log` to hack around the fact that there is not a proper console library yet. There will be, probably with cross-platform colors and all that too. So I'd say keep the implementation as is, but wrap it up as a task such that it runs when the task is run.

Based on that, I think [this code](https://github.com/project-fuzzball/test/blob/e294e6a62099eca1a3caf34f70b3a503904ca750/examples/LogRunnerExample.elm#L21-L28) should be pretty different. With a threaded implementation, things may be "out working" for quite some time (whoo tasks!) so you would want to hear about the results only when it's done, and display something else in the meantime. So I'd expect this task to be run in `init`. Once I am done with the server-side rendering stuff, there should be a function in `core` that produces `Program` values without an `html` dependency, so I'd plan for that reality, even if it means having a `elm-lang/virtual-dom` dependency temporarily.